src/plus/gk/account/subscriptionService.ts

```diff
@@ -75,6 +76,7 @@ export class SubscriptionService implements Disposable {
 
 	private _disposable: Disposable;
 	private _subscription!: Subscription;
+	private _checkinData: () => Promise<GKCheckInResponse | undefined>;
```

Comment:
Why a function vs `Promise<GKCheckInResponse | undefined>`?

Reply:
So that it doesn't get called until needed. When I set it up as a promise it was getting called in the constructor.